openshift-knative-operator/pkg/serving/extension_test.go

```diff
@@ -134,10 +292,24 @@ func TestReconcile(t *testing.T) {
 			if !cmp.Equal(ks, c.expected, opt) {
 				t.Errorf("Got = %v, want: %v, diff:\n%s", ks, c.expected, cmp.Diff(ks, c.expected, opt))
 			}
+			ns, err := kube.CoreV1().Namespaces().Get(context.Background(), ks.Namespace, metav1.GetOptions{})
+			if err != nil {
+				t.Errorf("Namespace %s not found %w", ns, err)
+			}
+			if len(ns.Labels) != 1 && ns.Labels[monitoring.EnableMonitoringLabel] != strconv.FormatBool(shouldEnableMonitoring) {
```

Comment:
I'd drop the explicit len check here. The explicit check for the name itself is fine and it'll be more extensible in the future if we ever add more labels.